python/cudf/cudf/core/column/string.py

```diff
@@ -958,6 +955,7 @@ def nvcategory(self):
             self._nvcategory = nvc.from_strings(self.nvstrings)
         return self._nvcategory
 
+    # TODO: Remove these once NVStrings is fully deprecated / removed
```

Comment:
idea: open a PR and use `# TODO(<PR number>): Remove these...`, so it's easy to search for later. Not sure how helpful that will be, depending on the granularity of the task.

Reply:
Not following exactly what you're proposing. It should be easy enough to just do a find of `nvstrings` or `nvcategory` to pluck all of these out in the future though.

Reply:
In general, if I'm making "todo" comments, I like to force myself to open a PR first, and put the pr number in the todo. `TODO(cudf/5342): somethingorother`. That has two benefits: 1) you know the PR got opened before you finished writing the "todo", and 2) the todos can be found more easily by pr number, and vice-versa.

It's less useful in broad-phase refactor/deletions/cleanup, and more useful when the task is more obscure/specific.

Reply:
Got it, yea if it was something smaller / more immediately actionable that would make complete sense. Given this would be a large scale rip out that's still going to take a bit more time and community coordination / announcement I don't think it would be helpful yet.